types/shippo/index.d.ts

```diff
@@ -1,65 +1,501 @@
-// Type definitions for shippo 1.5
+// Type definitions for shippo 1.7
 // Project: https://github.com/goshippo/shippo-node-client
 // Definitions by: Saiichi <https://github.com/saiichihashimoto>
+//                 James Lismore <https://github.com/jlismore>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.1
 
 declare namespace Shippo {
+    type USPSServiceLevel =
```

Comment:
It _feels_ like these big enums are likely to fall out of date; is there a reason to not leave them as `string` if they come from some API that can change (and provide users with a false sense of security as to the closed-ness of the set of valid results)?

Reply:
Having them this way gives autocomplete which saves me having to always consult the docs to check for valid options, but I see you're point about maintainability. I can add `| string` to all these types (similar to something like [csstypes](https://unpkg.com/csstype@3.1.0/index.d.ts)) to allow all strings but also provide autocomplete.

Reply:
That might not end up working in newer versions of typescript, depending on whether or not the union is converted just into string, but yeah, I guess a few definitions do that.

Reply:
If I add just `| string` then the compiler collapses the whole type down to just `string`. Other packages instead add `| (string & {})` which is what I did